### ISSUE TITLE
Add driver function for installing a study addon from a URL

### ISSUE THREAD

**Issue by Osmose** [mozilla/normandy#748]
Opt-out studies will be given a URL to an XPI file to install. The action will need to be able to hand this URL to the driver for installation. 

In order to power the listing page for current and past studies, we'll need to store some information about the study and track when it is removed, so I suggest a study-specific API:

```js
await normandy.studies.install('https://example.com/addon.xpi');
```

I'll file a separate issue for tracking the info about the study.

@crankycoder Thoughts?

**Comment by Osmose** [comment]
We'll need an API for uninstalling add-ons as well, so that we can remove studies that have been rolled back on the server.

**Comment by Osmose** [comment]
This landed in #778.
